<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="orzel" name="pl.edu.agh.cs.kraksim.routing.rulesprediction.statistics.DurationLevelArchiveTest" tests="3" failures="0" timestamp="10 Jun 2012 00:12:45 GMT" time="0,015" errors="1">
  <testcase name="testAddThreePeriods" time="0,000" classname="pl.edu.agh.cs.kraksim.routing.rulesprediction.statistics.DurationLevelArchiveTest"/>
  <testcase name="testAddTwoPeriods" time="0,000" classname="pl.edu.agh.cs.kraksim.routing.rulesprediction.statistics.DurationLevelArchiveTest">
    <error type="java.lang.AssertionError" message="expected:&lt;false&gt; but was:&lt;true&gt;">
      <![CDATA[java.lang.AssertionError: expected:<false> but was:<true>
	at org.testng.Assert.fail(Assert.java:89)
	at org.testng.Assert.failNotEquals(Assert.java:489)
	at org.testng.Assert.assertEquals(Assert.java:118)
	at org.testng.Assert.assertEquals(Assert.java:281)
	at org.testng.Assert.assertEquals(Assert.java:291)
	at pl.edu.agh.cs.kraksim.routing.rulesprediction.statistics.DurationLevelArchiveTest.testAddTwoPeriods(DurationLevelArchiveTest.java:37)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:57)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:601)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:76)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:673)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:846)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1170)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
	at org.testng.TestRunner.runWorkers(TestRunner.java:1147)
	at org.testng.TestRunner.privateRun(TestRunner.java:749)
	at org.testng.TestRunner.run(TestRunner.java:600)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:317)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:312)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:274)
	at org.testng.SuiteRunner.run(SuiteRunner.java:223)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1049)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:974)
	at org.testng.TestNG.run(TestNG.java:905)
	at org.testng.remote.RemoteTestNG.run(RemoteTestNG.java:111)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:203)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:174)
]]>
    </error>
  </testcase>
  <testcase name="testAddOnePeriod" time="0,015" classname="pl.edu.agh.cs.kraksim.routing.rulesprediction.statistics.DurationLevelArchiveTest"/>
</testsuite>
